Purchases.podspec

```diff
@@ -17,6 +17,8 @@ Pod::Spec.new do |s|
 
   s.ios.deployment_target = '9.0'
   s.osx.deployment_target = '10.12'
+  s.watchos.deployment_target = '6.2'
+  s.tvos.deployment_target = '13.4'
```

Comment:
don't we support older versions?

Reply:
I was honestly not sure about this. Since there wasn't a test app and tvOS wasn't a part of the target device families, I didn't even know whether it compiled on tvOS before this. 
I meant to test further on tvOS but I got distracted with other stuff. 
I just tried a new tvOS app on 3.0.4 and it does compile and seems to work. I don't have a physical apple tv so I can't get much further than that, unfortunately. 
I'll change the target to tvOS 9 since it's the first one that's compatible with StoreKit. Maybe we could sync up later about how best to test this?

Reply:
chatted about this on slack. Since we currently don't specify the target, nothing stops you right now from using purchases-ios with tvOS 9+. It won't compile with anything before that because StoreKit wasn't included in tvOS until v9. 
we're going to go with `deployment_target` = 9 to reflect this, so that at least we have an explicit value to reference.